sgrep.py

```diff
@@ -399,21 +400,18 @@ def flatten_rule_patterns(all_rules):
 
 def adjust_for_docker():
     # change into this folder so that all paths are relative to it
-    if Path(REPO_HOME_DOCKER).exists():
+    if IN_DOCKER:
+        if not Path(REPO_HOME_DOCKER).exists():
+            print_error_exit(
+                f"you are running sgrep in docker, but you forgot to mount the current directory in Docker: missing: -v $(pwd):{REPO_HOME_DOCKER}"
+            )
         os.chdir(REPO_HOME_DOCKER)
     elif Path(PRE_COMMIT_SRC_DOCKER).exists():
         os.chdir(PRE_COMMIT_SRC_DOCKER)
 
 
 def get_base_path() -> Path:
-    docker_folder = Path(REPO_HOME_DOCKER)
-    pre_commit_folder = Path(PRE_COMMIT_SRC_DOCKER)
-    if docker_folder.exists():
-        return docker_folder
-    elif pre_commit_folder.exists():
-        return pre_commit_folder
-    else:
-        return Path(".")
+    return Path(".")
```

Comment:
Does this mean that the tool assumes it's running in the code directory?

Reply:
It just means that the default directory is `.`. If you provide a relative path, it will be relative to `.`; if you provide an absolute path, that will still be honored.